resources/lib/common/kodiops.py

```diff
@@ -37,7 +40,7 @@ def json_rpc(method, params=None):
                     'params': params or {}}
     request = json.dumps(request_data)
     debug('Executing JSON-RPC: {}'.format(request))
-    raw_response = unicode(xbmc.executeJSONRPC(request), 'utf-8')
+    raw_response = text(xbmc.executeJSONRPC(request)).encode('utf-8')
```

Comment:
With the series "3%" exported, I can't start an episode. I always get a  UnicodeDecodeError: 'ascii' codec can't decode byte 0xc3 in position 1013: ordinal not in range(128). This is not related to any accentuated chars, it does the same with series with no accent or anything special in the name of the series or the episode.

I don't know how to fix it, and I fear all text() functions are suffering the issue :/

See here for the first error in json_rpc() https://paste.kodi.tv/vitenogebu.kodi
And here for the same kind of error in _decrypt_chunks() https://paste.kodi.tv/olinafikon.kodi

Till this error exists, I can't go further. I'm sorry I can't help more.

Reply:
@Smeulf You have an actual Kodi with Python 3 support that works?

Reply:
No it's Python 2 (Kodi 18.2, "prod" Raspberry Pi)

Reply:
But that's not the exact same code you proposed in your PR, your's was `raw_response = text(xbmc.executeJSONRPC(request), 'utf-8')`

Reply:
yes is not same becouse the  `text(xbmc.executeJSONRPC(request), 'utf-8')` do not work with py2 str do not have parameters...

Reply:
This works fine on Python 2:
```python
from builtins import str as text
print("%r" % text(u"wieërs", "utf-8"))
```
Returns `'wie\xc3\xabrs'`

Reply:
text is only a reference to str (in py2) look to the imports